azure-iot-device/azure/iot/device/common/pipeline/pipeline_exceptions.py

```diff
@@ -35,6 +29,12 @@ class PipelineTimeoutError(PipelineException):
 
 
 class PipelineError(PipelineException):
-    """Error in Pipeline"""
+    """Error in pipeline"""
+
+    pass
+
+
+class PipelineConfigurationError(PipelineException):
```

Comment:
>PipelineConfigurationError [](start = 6, length = 26)

A PipelineConfigurationError is not transient.  a PipelineError _could_ be transient.